scripts/tests.sh

```diff
@@ -39,60 +39,65 @@ then
     fi
     testargs_no_opt="--logger=JUNIT,test_suite,$2/no_opt.xml"
     testargs_opt="--logger=JUNIT,test_suite,$2/opt.xml"
+    shift
+    shift
```

Comment:
Why do you need to remove argv's here?

Reply:
Another try that was since removed did some further processing of following arguments.